slack-archive/src/main/java/juja/microservices/slack/archive/Application.java

```diff
@@ -1,12 +1,43 @@
 package juja.microservices.slack.archive;
 
+import org.apache.http.impl.client.HttpClients;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.List;
 
 @SpringBootApplication
+@EnableScheduling
```

Comment:
Please, move @EnableScheduling to SchedulerConfig class
It will be better

Reply:
done

Reply:
done